crypto/asn1/asn1_gen.c

```diff
@@ -564,7 +564,7 @@ static int asn1_str2tag(const char *tagstr, int len)
 
     tntmp = tnst;
     for (i = 0; i < OSSL_NELEM(tnst); i++, tntmp++) {
-        if ((len == tntmp->len) && (strncmp(tntmp->strnam, tagstr, len) == 0))
```

Comment:
windows build failure is relevant..

#include "e_os.h"                /* strncasecmp() */